src/Pages/category/components/CategoryContent/CategoryCarousel.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Carousel } from 'react-responsive-carousel';
+import { CAROUSEL_DATA } from './mockData';
+
+export default function CategoryCarousel() {
+    return (
+        <CarouselWrap>
+            <Carousel
+                showThumbs={false}
+                autoPlay={true}
+                interval={3000}
+                infiniteLoop={true}
+                showStatus={false}
+            >
+                {CAROUSEL_DATA.map((item, index) => {
+                    const { url } = item;
+                    return (
+                        <CarouselImgBox key={index}>
+                            <img src={url} alt="배너 화면" />
```

Comment:
```suggestion
                {CAROUSEL_DATA.map(({ url }, index) => (
                        <CarouselImgBox key={index}>
                            <img src={url} alt="배너 화면" />
```
함수 인자 자리에서 직접 구조분해 할당이 가능합니다